### DISCUSSION TITLE
GitHub Externship Page Link broken.

### DISCUSSION THREAD

**Discussion by manavmodi22** [github/india#65] [Students]
Github India has a readme file about the recently announced Externships. It mentions a site but the site isn't working
[https://github.com/github/india/blob/main/Students/EXTERNSHIPS.md](https://github.com/github/india/blob/main/Students/EXTERNSHIPS.md)
![image](https://user-images.githubusercontent.com/55242552/112609481-947a6600-8e41-11eb-84d0-d3336815dc49.png)

**Comment by aktech** [comment]
The url you have entered is incorrect. It works for me.

**Reply by manavmodi22** [reply]
Try accessing the link through the Readme page. Also if it works for you please add a comment with the link as well!

**Reply by aktech** [reply]
I tried the link you shared: https://github.com/github/india/blob/main/Students/EXTERNSHIPS.md

and it works for me.

And also from the README, where it says **check out the program page**
<img width="871" alt="Screenshot 2021-03-26 at 09 44 33" src="https://user-images.githubusercontent.com/5647941/112613324-e0b0b080-8e17-11eb-95c2-60c63a6122ef.png">

**Reply by msupernaut** [reply]
Thanks for pointing it out @manavmodi22 , we are fixing the glitch !!

**Reply by manavmodi22** [reply]
My reference was to the " program page here " text. That is what is showing the error page.

![image](https://user-images.githubusercontent.com/55242552/112613508-2edca880-8e46-11eb-8b7a-2418d130fc4d.png)

**Reply by aktech** [reply]
Ah, I see.

**Reply by msupernaut** [reply]
Fixed !
